docker/qgis/Dockerfile

```
@@ -0,0 +1,40 @@
+# For finding latest versions of the base image see
+# https://github.com/SwissDataScienceCenter/renkulab-docker
+ARG BASE_IMAGE=renku/renkulab-vnc:latest
+FROM ${BASE_IMAGE}
+
+USER root
+
+# Install qGIS
+RUN apt-get update \
+    && apt-get install -y -q --no-install-recommends \
+        gnupg \
+        software-properties-common \
+        libqt5gui5 \
+    && apt-get autoremove --purge \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
+    && rm -rf /tmp/*
+
+RUN wget -qO - https://qgis.org/downloads/qgis-2021.gpg.key | sudo gpg --no-default-keyring --keyring gnupg-ring:/etc/apt/trusted.gpg.d/qgis-archive.gpg --import \
```

running as `root`, don't need `sudo`

```suggestion
RUN wget -qO - https://qgis.org/downloads/qgis-2021.gpg.key | gpg --no-default-keyring --keyring gnupg-ring:/etc/apt/trusted.gpg.d/qgis-archive.gpg --import \
```